### PR TITLE
Httpclient and interceptor.

### PR DESCRIPTION
**Pull request by jacol12345**
__Story / Bug id:__ https://github.com/CodeForPoznan/volontulo/issues/744
__Reference person:__ @jacol12345
__Description:__

Use HttpClient. Add interceptor.

### CODE REVIEW

File: frontend/src/app/offers/offers.service.ts [hunk 1]
```diff
@@ -10,11 +11,10 @@ import { Offer } from './offers.model';
 export class OffersService {
   private url = `${environment.apiRoot}/offers/`;
 
-  constructor (private http: Http) { }
+  constructor (private http: HttpClient) { }
 
-  getOffers() {
-    return this.http.get(this.url, { withCredentials: true } )
-      .map((res: Response) => res.json());
+  getOffers(): Observable<Offer[]>{
```
Comment (radoslawroszkowiak) [inline frontend/src/app/offers/offers.service.ts]: missing whitespace ;)

**Comment by radoslawroszkowiak** [comment]
I did not run / test it yet, but LGTM :+1:

**Comment by jacol12345** [comment]
@radoslawroszkowiak Linting issues fixed.

**Comment by magul** [comment]
@jacol12345 can You resolve conficts here?

**Pull request closed without merging**